packages/microsoft-reactnative-sampleapps/index.windows.js

```diff
@@ -310,6 +310,16 @@ class SampleApp extends Component {
     log(`SampleApp.onLabelChangedCustomUserControlCpp("${label}")`);
   }
 
+  onReloadSampleModuleCS() {
```

Comment:
Should we add a case for reloading to the integration tests?

Reply:
Could you point me at an example how to do it?

---
In reply to: [516257677](https://github.com/microsoft/react-native-windows/pull/6408#discussion_r516257677) [](ancestors = 516257677)

Reply:
Here are some tests that Danny added for interacting with CS modules from JS https://github.com/microsoft/react-native-windows/blob/master/packages/IntegrationTest/tests/InteropCSModuleTests.tsx

I think adding this style of test might be tricky though, since state would be lost when we reload the instance.

Reply:
In that case I can add them later in a different PR.